itou/job_applications/models.py

```diff
@@ -606,6 +606,39 @@ def display_sender_kind(self):
                 kind = "Prescripteur habilité"
         return kind
 
+    @property
+    def is_waiting_for_FS(self):
+        """
+        Returns FS creation status if EmployeeRecord does not exit.
+        Used in admin page, dynamic read-only field "employee_record_status"
+
+        Rules on Job Applications
+        x job application must be accepted
+        x job starts after settings.EMPLOYEE_RECORD_FEATURE_AVAILABILITY_DATE
+        x approval must not be refused : approval_manually_refused_at date is None
+        x hiring with approval : hiring_without_approval is False
+
+        Rule on Employee Record
+        x you can only create *one* employee record for a given asp_id / approval pair
+        """
```

Comment:
AMHA, les conditions nécessaires et suffisantes sont : 
- pas d'existence d'une fiche pour une paire PASS IAE / `asp_id`,
- présence d'un PASS IAE valide,
- la date de début du contrat postérieure à la date de mise en production de la fonctionnalité
- la SIAE cible est eligible, voir https://github.com/betagouv/itou/blob/eaf333e806680cc5f341846fb43e6d749a2f8d21/itou/siaes/models.py#L423

Reply:
meme humbles, ce sont bien vos avis que je recherche :D 
je verifie avec Zo sur les regles que tu rappeles. Merci Fred

Reply:
il me manque un controle sur la SIAE eligilble : merci

Reply:
j'ai laissé un controle sur `not self.hiring_without_approval` au cas où le demandeur aurait un pass valid mais que la SIAE recruterait sans pass.
peut-être un cas "chauve-souris". Dis moi si ça fait sens pour toi …